12-buster-slim/Dockerfile

```diff
@@ -26,7 +26,10 @@ RUN apt-get update -qq \
     && mkdir -p $NODE_MODULES_PATH && chown $SERVICE_USER:$SERVICE_USER $NODE_MODULES_PATH
 
 # Add a strict security policy for Imagemagick (note: Imagemagick 7 is currently not available in Debian)
-COPY imagemagick-policy.xml /etc/ImageMagick-6/policy.xml
+COPY policies/imagemagick-policy-6.xml /etc/ImageMagick-6/policy.xml
+
+# Add custom retry script
+COPY scripts/retry /usr/local/bin
```

Comment:
If we don't have a shared directory, we would also need to copy this over to every directory. We could also use `ADD` instead and pull in an [external script](https://raw.githubusercontent.com/maxiberta/bash-retry/a30e1077caf936b2f2900886011e0a446a8edb28/retry.sh), using a permalink to reduce supply chain issues

Reply:
Do permalinks persist beyond somebody deleting a repo?